drivers/watchdog/wdt_nrf.c

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2018, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrfx_wdt.h>
+#include <nrf_clock.h>
+#include <clock_control.h>
+#include <watchdog.h>
+
+#define SYS_LOG_DOMAIN "WDT NRF"
+#define SYS_LOG_LEVEL CONFIG_SYS_LOG_WDT_LEVEL
+#include <logging/sys_log.h>
+
+#define NRF_WDT_CHANNELS_NUMBER 8
+
+struct wdt_nrf_dev_data {
+	void (*cb)(struct device *dev, void *handle);
+};
```

Comment:
Is this still needed? It looks like remains from an earlier version.
See also line [183](https://github.com/zephyrproject-rtos/zephyr/pull/7007/files#diff-55344f9e3b6305124e27f1ad2aea237bR183).